addons/mail/static/src/discuss/call/common/device_select.js

```diff
@@ -18,9 +18,11 @@ export class DeviceSelect extends Component {
     setup() {
         super.setup();
         this.store = useService("mail.store");
+        this.notification = useService("notification");
```

Comment:
ok (seems this was removed by mistake?).
Note that using toaster is not exactly what we want: instead we want to show the problem next to buttons.
This will be done in another task eventually.